hibernation-setup-tool.c

```diff
@@ -1468,6 +1468,10 @@ static int recursive_rmdir_cb(const char *fpath, const struct stat *st, int type
     case FTW_D: /* directory */
         return rmdir(fpath) == 0 ? FTW_CONTINUE : FTW_STOP;
 
+    case FTW_DP: 
+        log_info("Removing %s directory. Its subdirectories and subfiles have been visited.", fpath);
+        return rmdir(fpath) == 0 ? FTW_CONTINUE : FTW_STOP;
+    
```

Comment:
Given flag FTW_DEPTH performs postorder traversal, after removing subfiles and subdirectories, we have to remove the root directory this function was called from itself. We do that by adding this case here.